src/SymptomHistory/index.tsx

```diff
@@ -10,14 +17,30 @@ import { SymptomEntry } from "./symptomHistory"
 import { Text, StatusBar } from "../components"
 import { useStatusBarEffect } from "../navigation"
 import SymptomEntryListItem from "./SymptomEntryListItem"
+import Formatter from "./Formatter"
 
-import { Typography, Colors, Spacing } from "../styles"
+import { Colors, Spacing, Typography, Iconography } from "../styles"
+import { Icons } from "../assets"
 
 const SymptomHistory: FunctionComponent = () => {
   useStatusBarEffect("dark-content", Colors.primaryLightBackground)
   const { t } = useTranslation()
   const { symptomHistory } = useSymptomHistoryContext()
 
+  const handleOnPressShareHistory = async () => {
+    const message = Formatter.forSharing(t, symptomHistory)
+
+    if (message) {
+      try {
```

Comment:
Could the `const message` go in the `try` block? Not sure if that works or is good practice, but maybe we could get rid of the `if (message)`.